sdk/logs/src/main/java/io/opentelemetry/sdk/logs/internal/LoggerConfig.java

```diff
@@ -62,6 +69,86 @@ public static ScopeConfiguratorBuilder<LoggerConfig> configuratorBuilder() {
 
   LoggerConfig() {}
 
+  /**
+   * Builder for {@link LoggerConfig}.
+   *
+   * <p>This class is internal and experimental. Its APIs are unstable and can change at any time.
+   * Its APIs (or a version of them) may be promoted to the public stable API in the future, but no
+   * guarantees are made.
+   */
+  public static final class Builder {
+    private boolean enabled = true;
+    private int minimumSeverity = 0;
+    private boolean traceBased = false;
+
+    private Builder() {}
+
+    /**
+     * Sets whether the logger is enabled.
+     *
+     * @param enabled whether the logger is enabled
+     * @return this builder
+     */
+    public Builder setEnabled(boolean enabled) {
+      this.enabled = enabled;
+      return this;
+    }
+
+    /**
+     * Sets the minimum severity level for log records to be processed.
+     *
+     * <p>Log records with a severity number less than this value will be dropped. Log records
+     * without a specified severity are not affected by this setting.
+     *
+     * @param minimumSeverity minimum severity level for log records to be processed
+     * @return this builder
+     */
+    public Builder setMinimumSeverity(int minimumSeverity) {
```

Comment:
Did you consider `io.opentelemetry.api.logs.Severity` here? If yes, why did you choose against it?